Formula/openresty.rb

```diff
@@ -69,6 +68,10 @@ def install
     system "make"
     system "make", "install"
   end
+  
```

Comment:
Error: 1 problem in 1 formula
homebrew/nginx/openresty:
  * 71: Trailing whitespace was found